scripts/travis/configure_dev.sh

```diff
@@ -1,5 +1,56 @@
 #!/usr/bin/env bash
 
+# keep script execution on errors
+set +e
+
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
+OS=$(${SCRIPTPATH}/../ostype.sh)
+ARCH=$(${SCRIPTPATH}/../archtype.sh)
+
+if [ "${OS}" = "linux" ]; then
```

Comment:
Seems like there should be a better way to do this. Can we set the `ARCH` to `armv61` and use that variable?

Reply:
I'm confused about the context of your comment. Please stop by, and let's discuss that.